holmes/config.py

```diff
@@ -477,8 +479,8 @@ def _get_llm(self, model_key: Optional[str] = None, tracer=None) -> "LLM":
             )
             api_key = model_params.pop("api_key", api_key)
             model = model_params.pop("model", model)
-
-        from holmes.core.llm import DefaultLLM
```

Comment:
I think that this is intentionally done by @aantn to speed up the holmes cli startup.